CGCharter-1727386911.html

```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
+        a chair, for any reason, that participant must first privately
+        communicate with one or more of the other chairs and/or W3C contact
+        their desire and reason for doing so. Those chairs must give the
+        participant an opportunity to discuss their concerns with a goal of
+        resolving them. If, after 30 days, the participant feels their issues
+        have not been adequately addressed, they may then escalate to a public
+        call for a no confidence vote (without having to disclose with whom they
+        have been discussing the matter). If, after 60 days, the participant has
+        not made a call for a no confidence vote, the matter should be
+        considered dropped; any further attempts to remove the chair in question
+        must begin with a new round of private communication.
+      </li>
+      <li>
+        A public call for no confidence must be announced to the group email
+        list stating the name of the chair subject to the recall. This
+        announcement must come from the participant who initiated the private
+        communication with the chairs to discuss their concerns. At least two
+        other supporting participants MUST reply to that email on the public
+        list within 48 hours to confirm their support for a no-confidence vote
+        of the chair in question.
+      </li>
+      <li>
+        The other chairs must acknowledge the call for no confidence within 7
+        days of all three participants declaring their support.
+      </li>
+      <li>
+        UNLESS the chair in which no-confidence has been expressed voluntarily
+        offboards and opts out of the session on the mailing list, and within 30
+        days of a call for no confidence, the other chairs MUST hold a
+        conference call at which the parties seeking no confidence will have an
+        opportunity to present their case and participants of the group will be
+        able to ask clarifying questions. The chair in question SHALL NOT
+        moderate this call. They will, however, have equal time to respond
+        during that same call to the case made against them.
+      </li>
+      <li>
+        During the week following this conference call, participants may cast
+        votes in favor or against the recall by posting to the email list.
+      </li>
+      <li>
+        If affirmative votes cast that week (in favor of recall) comprise
+        greater than two-thirds of the total votes cast, then the chair in
+        question is removed and the seat shall be treated as an interim vacancy.
+      </li>
+      <li>
+        Participants are not required to vote. Abstentions may be recorded; such
+        abstentions shall not count towards the total number of votes when
+        calculating the two-thirds majority.
+      </li>
+      <li>          
+        A Chair who has been removed may stand for re-election.
```

Comment:
```suggestion
        A Chair who has been removed may stand for reelection.
```

(normalizing to above)